edi_storage_oca/__manifest__.py

```diff
@@ -7,7 +7,7 @@
     "summary": """
     Base module to allow exchanging files via storage backend (eg: SFTP).
     """,
-    "version": "17.0.1.1.2",
+    "version": "18.0.1.1.2",
```

Comment:
Usually, we clean the versions during migration.
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-18.0#tasks-to-do-in-the-migration
Bump module version to 18.0.1.0.0.